PayPal/src/main/java/com/braintreepayments/api/paypal/vaultedit/PayPalVaultEditResponse.kt

```diff
@@ -0,0 +1,46 @@
+package com.braintreepayments.api.paypal.vaultedit
+
+import com.braintreepayments.api.BrowserSwitchOptions
+import com.braintreepayments.api.core.ExperimentalBetaApi
+
+@ExperimentalBetaApi
+sealed class PayPalVaultEditResponse {
+
+    /**
+     * The PayPal vault edit flow completed successfully.
+     *
+     * @property riskCorrelationId This ID is used to link subsequent retry attempts if payment is declined
+     */
+    class ReadyToLaunch internal constructor(
+        val riskCorrelationId: String,
+        val response: EditFIAgreementSetup,
```

Comment:
Is this property used at all? It should be internal if needed, but it doesn't look like it's used